easybuild/easyblocks/l/llvm.py

```diff
@@ -369,18 +369,19 @@ def __init__(self, *args, **kwargs):
         if LooseVersion(self.version) < LooseVersion('16'):
             self.general_opts['LLVM_INCLUDE_GO_TESTS'] = 'OFF'
 
-        # Sysroot
-        self.sysroot = build_option('sysroot')
-        if self.sysroot:
-            if LooseVersion(self.version) < LooseVersion('19'):
-                raise EasyBuildError("Using sysroot is not supported by EasyBuild for LLVM < 19")
-            self.general_opts['DEFAULT_SYSROOT'] = self.sysroot
-            self.general_opts['CMAKE_SYSROOT'] = self.sysroot
-            self._set_dynamic_linker()
-            trace_msg(f"Using '{self.dynamic_linker}' as dynamic linker from sysroot {self.sysroot}")
+        self.log.info("Final projects to build: %s", ', '.join(self.final_projects))
+        self.log.info("Final runtimes to build: %s", ', '.join(self.final_runtimes))
 
-        self.ignore_patterns = self.cfg['test_suite_ignore_patterns'] or []
+        self._cmakeopts = {}
+        self._cfgopts = list(filter(None, self.cfg.get('configopts', '').split()))
```

Comment:
This is still (also) done on lines 480-481?

Reply:
Missed those 2 lines were also duplicated by the rebase